cmd/kube-proxy/app/server_others_test.go

```diff
@@ -142,6 +145,14 @@ func Test_getProxyMode(t *testing.T) {
 			kernelCompat:  true,
 			expected:      proxyModeIPTables,
 		},
+		{ // flag says ipvs, ipset version ok, kernel modules installed for sed scheduler
```

Comment:
Would it be possible to add an error case here too where we fall back to another proxy mode when the desired scheduler module doesn't exist?